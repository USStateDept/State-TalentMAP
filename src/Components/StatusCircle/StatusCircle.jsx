import FontAwesome from 'react-fontawesome';

const StatusCircle = () => (
  <span className="status">
    <FontAwesome name="circle" />
  </span>
);

export default StatusCircle;
