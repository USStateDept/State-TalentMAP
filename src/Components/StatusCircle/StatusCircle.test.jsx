import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import StatusCircle from './StatusCircle';

describe('StatusCircle', () => {
  it('is defined', () => {
    const wrapper = shallow(<StatusCircle />);
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<StatusCircle />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
