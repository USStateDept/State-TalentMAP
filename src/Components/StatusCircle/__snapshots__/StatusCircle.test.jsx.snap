// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusCircle matches snapshot 1`] = `
<span
  className="status"
>
  <FontAwesome
    name="circle"
  />
</span>
`;
